flink-runtime/src/main/java/org/apache/flink/runtime/io/network/NetworkSequenceViewReader.java

```diff
@@ -40,6 +45,30 @@ void requestSubpartitionView(
             int subPartitionIndex)
             throws IOException;
 
+    /**
+     * The {@link PartitionRequestNotifier} notify and process the downstream task's partition request.
+     *
+     * @param partition the result partition
+     * @param subPartitionIndex the sub partition index
+     * @throws IOException the thrown exception
+     */
+    void requestSubpartitionView(ResultPartition partition, int subPartitionIndex) throws IOException;
```

Comment:
I think those origin methods should be renamed and reimplemented, same as others.

Reply:
Done